client/src/pages/ChefProfile.js

```diff
@@ -1,16 +1,159 @@
+
+  useEffect(() => {
+    const getProfileAndRecipes = async () => {
+      const res = await axios.get(`/api/chefProfiles/${userInfo._id}`);
+      if (res.data) {
+        setProfile(res.data.chefProfile);
+        getRecipesByChef(dispatch, res.data.chefProfile._id);
+      }
+    };
+    getProfileAndRecipes();
+  }, [dispatch, userInfo._id]);
+
+  const chefInfosAndRecipes = {
+    name: "Gordon Ramsey",
+    chefProfile: {
+      coverPictureUrl:
+        "https://media.gq.com/photos/5a03339d1a820c56cf27b8b3/16:9/pass/you're-gonna-love-pittsburgh-0917-gq-matr01-02.jpg",
+      location: "Toronto, ON",
+      bio:
+        "Gordon Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+      profilePictureUrl:
+        "https://i1.sndcdn.com/avatars-000624434067-r5kehg-t500x500.jpg",
+      cuisineTags: ["european", "american", "asian"],
+    },
+  };
+  //   recipes: [
+  //     {
+  //       name: "lorem ipsum",
+  //       pictureUrl:
+  //         "https://i2-prod.mirror.co.uk/incoming/article9911973.ece/ALTERNATES/s615b/170125_AllStarLanes_Pancakes_Banoffee.jpg",
+  //       price: "$3.97",
+  //       ingredients: [
+  //         "rice",
+  //         "avocado",
+  //         "crab",
+  //         "cucumber",
+  //         "wasabi",
+  //         "rice",
+  //         "vinegar",
+  //         "soy sauce",
+  //         "salt",
+  //         "sugar",
+  //       ],
+  //       requiredStuffs: ["pan", "oil", "kitchen table"],
+  //       portionDescription: "meal for 2",
+  //       cuisineTags: ["european"],
+  //     },
+  //     {
+  //       name: "lorem ipsum",
+  //       pictureUrl:
+  //         "https://i2-prod.mirror.co.uk/incoming/article9911973.ece/ALTERNATES/s615b/170125_AllStarLanes_Pancakes_Banoffee.jpg",
+  //       price: "$3.97",
+  //       ingredients: [
+  //         "rice",
+  //         "avocado",
+  //         "crab",
+  //         "cucumber",
+  //         "wasabi",
+  //         "rice",
+  //         "vinegar",
+  //         "soy sauce",
+  //         "salt",
+  //         "sugar",
+  //       ],
+  //       requiredStuffs: ["pan", "oil", "kitchen table"],
+  //       portionDescription: "meal for 2",
+  //       cuisineTags: ["european"],
+  //     },
+  //     {
+  //       name: "lorem ipsum",
+  //       pictureUrl:
+  //         "https://i2-prod.mirror.co.uk/incoming/article9911973.ece/ALTERNATES/s615b/170125_AllStarLanes_Pancakes_Banoffee.jpg",
+  //       price: "$3.97",
+  //       ingredients: [
+  //         "rice",
+  //         "avocado",
+  //         "crab",
+  //         "cucumber",
+  //         "wasabi",
+  //         "rice",
+  //         "vinegar",
+  //         "soy sauce",
+  //         "salt",
+  //         "sugar",
+  //       ],
+  //       requiredStuffs: ["pan", "oil", "kitchen table"],
+  //       portionDescription: "meal for 2",
+  //       cuisineTags: ["european"],
+  //     },
+  //   ],
+  // };
```

Comment:
Also for this commented out code, you can remove this if it's unused

Reply:
Thanks Dan, Sure I will delete them.